Makefile

```diff
@@ -72,6 +72,14 @@ install: $(EXES)
 uninstall: $(EXES)
 	rm -f $(foreach exe, $^, /usr/local/bin/$(notdir $(exe)))
 
+aws:
+	for f in o4/o4_pyforce.py build/o4 build/manifold build/gatling; do \
+	aws s3 cp $$f s3://sfdc-ansible/o4/$$(basename $$f); done
```

Comment:
Why does this need doubled dollar signs but line 73 doesn't?

Reply:
Because I’m stupid. Thanks! 👌

Reply:
At first I thought had messed up aws vs aws-staging. Stupid still holds though.

That's because this is a long `bash -c <make-file-line>` so it's interpreted by make first, turning double dollars to single dollars (a lot of systems in computer are like that. LOL!).

Line 73 otoh make is expanding the foreach and all the variable. And that's where stupid comes in, I should have used foreach both times, but I was too tired to make a pure solution.